lib/Types.ts

```diff
@@ -9,8 +8,8 @@ export type Bindings = Map<string, RDF.Term>;
  * @return {Bindings} The immutable bindings from the hash.
  * @constructor
  */
-export function Bindings(hash: Record<string, RDF.Term>): Bindings {
-  return Map(hash);
+export function BindingsFromHash(hash: Record<string, RDF.Term>): Bindings {
```

Comment:
This (`Bindings`) is apparently literally taken over from Comunica.
Not sure how strict the relationship to Comunica is with respect to this, but can you double-check if everything still works after this change?

In any case, I see no need to rename this function.

Reply:
Well, the linter doesn't like a function Bindings and a type Bindings to have the same name. it does seem like comunica has some problems with this PR tho. I should investigate that closer.

Reply:
Seems like `BindingsFromHash` Isn't used in comunica. `Bindings` however is (in the config object, within exists). So I just made that an immutable Map again.

Reply:
Ok, good!

Could you restore the implementation as well to what it was before?

And I would recommend keeping the name `Bindings` for consistency with Comunica. It's ok to add a linter exception for this.

Reply:
Okey!